packages/remix/package.json

```diff
@@ -70,11 +70,12 @@
     "fix": "eslint . --format stylish --fix",
     "lint": "eslint . --format stylish",
     "test": "yarn test:unit",
-    "test:integration": "run-s test:integration:v1 test:integration:v2",
+    "test:integration": "run-s test:integration:v1 test:integration:v2 test:integration:tracingIntegration",
     "test:integration:v1": "run-s test:integration:clean test:integration:prepare test:integration:client test:integration:server",
     "test:integration:v2": "export REMIX_VERSION=2 && run-s test:integration:v1",
+    "test:integration:tracingIntegration": "export TRACING_INTEGRATION=true && run-s test:integration:v2",
     "test:integration:ci": "run-s test:integration:clean test:integration:prepare test:integration:client:ci test:integration:server",
-    "test:integration:prepare": "(cd test/integration && yarn)",
+    "test:integration:prepare": "(cd test/integration && yarn install --force)",
```

Comment:
q: why do we need the `--force` flag?

Reply:
Nope, we don't need. It's a leftover from when I'd been yarn linking / unlinking a lot to test this. Removing.